insonmnia/locator/storage/libkv/storage.go

```diff
@@ -0,0 +1,75 @@
+package libkv
```

Comment:
ehal storage cherez storage, storage storage storage storage

Reply:
BTW, where do we use this one also?

Reply:
Could you please explain what's so wrong with it? It's just a file name inside a package. It's not a part of an API.

engine "github.com/sonm-io/core/insonmnia/locator/storage/libkv"

Apparently it's used in the application, particularly during application initialisation App:Init()
	storage, err := engine.NewStorage(a.conf.NodeTTL, kv)